nyc_taxis/challenges/default.json

```diff
@@ -511,7 +511,8 @@
           "operation": {
             "operation-type": "start-ml-datafeed",
             "datafeed-id": "{{ml_feed_id}}",
-            "end": "now"
+            "start": "2015-01-01T00:00:00",
+            "end": "2016-01-01T00:00:00"
```

Comment:
If these are the start/end of the data set, then this is fine by me :)

Reply:
Yes, they are :)